app/views/how-tos/make-first-prototype/branching.html

```diff
@@ -0,0 +1,76 @@
+
+{% set pageTitle = "Show different pages depending on user input (branching)" %}
+{% set next = { "title" : "Link your index page to your start page" , "link" : "link-index-page-start-page" } %}
+{% set prev = { "title" : "Let the user change their answers" , "link" : "let-user-change-answers" } %}
+
+{% extends 'how-tos/make-first-prototype/_BASE.njk' %}
+
+{% block makePrototype %}
+
+  <p>Our first question asks the user if they have had symptoms of magical powers in the last 30 days. We’re going to send them to an ‘ineligible’ page if answer anything other than yes. Sending users to different pages based on their input is called branching.</p>
+<p>To do this, we’re going to need an <code>{{examples[5].url}}.html</code> page, and some logic to decide when to send users there.</p>
```

Comment:
Referencing the urls by index number here is clever in reducing duplication, but perhaps kinda hard to read and maintain? eg adding an additional page to the array near the start would break everything?

I’d be tempted to hardcode all these URL references, and then rely on find-and-replace if any pages every get renamed in future?

Reply:
Yeah - this is probably a bit over-designed so will roll back for now, the main thing is that any changes are going to need a lot of checks just as the links are used in several places (similarly as to why I had real pages as a check just to make sure that everything worked).

Reply:
I've temporarily renamed the elements to things that are self-evident while we continue to test the pages - the observation I had from doing the translation is that since there is a lot of changing pages over multiple screens it's really hard to hardcode everything until it's settled (I found some extra juggling code that I'd missed!)